### PR TITLE
[YUNIKORN-2126] Fixing gocritic lint issues

### PR DESCRIPTION
**Pull request by makinyemi**
### What is this PR for?
Resolve gocritic lint issue:
1) commentFormatting: put a space between `//`
2) exitAfterDefer: log.Fatalf will exit, and `defer cancel()` will not run
3) use strings.ReplaceAll method in `strings.Replace(name, configs.DOT, configs.DotReplace, -1)`

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [x] - Refactoring

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2126

### How should this be tested?
I tested this locally with make test, but noticed there are no test for this package (2) 

### Screenshots (if appropriate)
n/a

### Questions:
n/a

### CODE REVIEW

File: cmd/schedulerclient/client.go [hunk 1]
```diff
@@ -35,6 +36,12 @@ const (
 )
 
 func main() {
+	if err := mainNoExit(); err != nil {
+		log.Fatalf("error: %v", err)
+	}
+}
+
+func mainNoExit() error {
```
Comment (pbacsko) [inline cmd/schedulerclient/client.go:44]: Nit: can we give this a different name, like `runApp()`? This sounds weird.

File: cmd/schedulerclient/client.go [hunk 2]
```diff
@@ -69,9 +76,9 @@ func main() {
 			log.Print("Send request")
 			time.Sleep(time.Millisecond * 100)
 		}
-		//if err := stream.CloseSend(); err != nil {
+		// if err := stream.CloseSend(); err != nil {
 		//   log.Println(err)
-		//}
+		// }
```
Comment (pbacsko) [inline cmd/schedulerclient/client.go:81]: This is unnecessary code, just remove it.

**Review by pbacsko — CHANGES_REQUESTED** [review]
Minor things.

File: cmd/schedulerclient/client.go (hunk 1 above)
Comment (makinyemi) [reply cmd/schedulerclient/client.go:44]: Sure thing

**Review by pbacsko — APPROVED** [review]
+1

**Pull request closed without merging**